### ISSUE TITLE
Part of NodeShape id is lost after toJsonSchema call

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#440]
Using `amf-client-js@3.1.7`
```js
const amf = require('amf-client-js')

const ramlStr = `
  #%RAML 1.0 Library
  types:
    Book:
      type: object
      properties:
        title: string
        author: string
`

async function main () {
  await amf.AMF.init()
  const model = await amf.AMF.raml10Parser().parseStringAsync(ramlStr)
  console.log('Before', model.declares[0].id)
  console.log('JSON', model.declares[0].toJsonSchema)
  console.log('After', model.declares[0].id)
}

main()
```

Outputs
```
Before http://a.ml/amf/default_document#/declarations/types/Book
JSON {
  "$schema": "http://json-schema.org/draft-04/schema#",
  "$ref": "#/definitions/Book",
  "definitions": {
    "Book": {
      "type": "object",
      "required": [
        "title",
        "author"
      ],
      "properties": {
        "title": {
          "type": "string"
        },
        "author": {
          "type": "string"
        }
      }
    }
  }
}

After null#/declarations/Book
```

Note the `After` id and how it lost `http://a.ml/amf/default_document` part. Same happens when parsing files.

**Comment by gutee** [comment]
Jira issue to track progress: https://www.mulesoft.org/jira/browse/APIMF-1530

**Comment by gutee** [comment]
This is fixed and will be available in the next release (currently 3.2.2 is out).
